pkg/handlers/notification.go

```diff
@@ -219,19 +218,15 @@ func RedisNotificationReceiver(conf config.SyncConfig) NotificationReceiverFunc
 			return nil, errors.New("sdk key not found")
 		}
 
-		redisSyncer, err := syncer.NewRedisSyncer(&zerolog.Logger{}, conf, sdkKey)
+		redisSyncer, err := syncer.NewSyncedNotificationCenter(ctx, sdkKey, conf)
```

Comment:
Can we remove "redis" from the name? Can we also change the function name in line 215 too? "Redis" should be removed from the name.